Python/optimizer_bytecodes.c

```diff
@@ -170,7 +170,7 @@ dummy_func(void) {
             if (temp == NULL) {
                 goto error;
             }
-            res = sym_new_const(ctx, temp);
+            OUT_OF_SPACE_IF_NULL(res = sym_new_const(ctx, temp));
```

Comment:
Maybe restore the old (incorrect) code for this and the next two chunks, since my PR gh-116062 will fix this better?